elliottlib/errata.py

```diff
@@ -764,3 +764,43 @@ def is_security_advisory(advisory):
 def is_advisory_impact_smaller_than(advisory, impact):
     i = [None] + constants.SECURITY_IMPACT
     return i.index(advisory.security_impact) < i.index(impact)
+
+
+def set_blocking_errata(target_advisory_id, blocking_advisory_id, blocking_state="SHIPPED_LIVE") -> dict:
```

Comment:
This is not a strong opinion, but I would like to rename it to `set_blocking_advisory`. We call it `advisory` everywhere in our code instead of confusing `erratum` or `errata`, and strictly speaking `errata` is the plural form of `erratum`. Apparently this API call doesn't accept multiple advisories...

Reply:
Done